packages/api/.env.example

```diff
@@ -15,6 +15,7 @@ DISABLE_EXTERNAL_API=false
 DATABASE_STATEMENT_TIMEOUT_MS=90000
 CONTRACT_VERIFICATION_API_URL=http://127.0.0.1:3070
 NETWORK_NAME=testnet-sepolia
+APP_URL=http://localhost:3010
```

Comment:
I think it makes sense to name it PRIVIDIUM_APP_URL following the same pattern as for other prividium-specific envs. It's just irrelevant for public instances as they might be used by many apps.